core/src/main/web/app/styles/theme/base-theme.scss

```diff
@@ -648,7 +648,7 @@
   }
 
   .CodeMirror-fullscreen {
-    background: $main-theme-panel-background;
+    background: $main-theme-panel-background !important;
```

Comment:
Is the style from the default theme overriding this? Is that why we need 'important'?